packages/react/src/TextArea/TextArea.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { createRef, useState } from 'react'
+import { TextArea } from './TextArea'
+import '@testing-library/jest-dom'
+
+describe('Text area', () => {
+  it('renders', () => {
+    render(<TextArea />)
+
+    const component = screen.getByRole('textbox')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    render(<TextArea />)
+
+    const component = screen.getByRole('textbox')
+
+    expect(component).toHaveClass('amsterdam-text-area')
+  })
+
+  it('renders an additional class name', () => {
+    render(<TextArea className="extra" />)
+
+    const component = screen.getByRole('textbox')
+
+    expect(component).toHaveClass('amsterdam-text-area extra')
+  })
+
```

Comment:
Can you add tests for the classes that are added by the `resize` and `cols` props?

Reply:
One of the classes added by `resize` is now tested, could you add the rest?